test/controllers/guitars_controller_test.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class GuitarsControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
```

Comment:
Rubocop complains on this line

Reply:
I'll have a look at it
Thanks @Ron-Lavi :)

Reply:
Fixed 👍🏼 resolving this conversation